### PR TITLE
Premium articles page

### PR DESCRIPTION
**Pull request by AlbertStjarne**
## PT:
https://www.pivotaltracker.com/story/show/161531527

## Included in PR
* Adds page where all premium articles are listed
* Adds page to read a specific premium article
* Adds link from main page to list of premium articles
* Rendering of page also if article/s are missing image
* Styling

## What we learnt
* How to render page if article/s are missing image

### CODE REVIEW

File: features/premium_listings_page.feature [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    | History   |
+    | Science   |
+
+    Given the following articles are in the database
+    | title              | content  | author | approved  | category    |
+    | Selfmade article   | Battles  | Snorre | true      | History     |
+    | WW3                | Robots   | Snorre | false     | History     |  
+
+  Scenario: Premium user can see list of premium content [Happy path]
+    Given I am signed in as "premium@example.com"
```
Comment (Rinbo) [inline features/premium_listings_page.feature:24]: Can be moved to background

File: features/premium_listings_page.feature [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    And I visit the main page
+    And I click "View all Premium Articles"
+    Then I am on the premium articles page
+    And I should see "Premium Articles page"
+    And I should see "Selfmade article"
+    And I should see "Battles"
+    
+  Scenario: Premium user reads full premium article [Happy path]
+    Given I am signed in as "premium@example.com"
```
Comment (Rinbo) [inline features/premium_listings_page.feature]: Can be moved to background

**Review by Rinbo — COMMENTED** [review]
Good job

File: features/premium_listings_page.feature (hunk 1 above)
Comment (AlbertStjarne) [reply features/premium_listings_page.feature:24]: Is now moved to background

File: features/premium_listings_page.feature (hunk 2 above)
Comment (AlbertStjarne) [reply features/premium_listings_page.feature]: Is now moved to background

**Pull request merged**